jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/Lambda.kt

```diff
@@ -25,6 +27,9 @@ data class LambdaFunction(
     val lastModified: String,
     val handler: String,
     val runtime: Runtime,
+    val envVariables: Map<String, String>?,
+    val timeout: Int,
```

Comment:
Other than `timeout`, there is another common setting `memory size`. I think it's better to add here as well.

Reply:
We dont support that in the UI yet. Will follow up with that